sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1932,6 +1932,13 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
     new UnsupportedOperationException(s"$nodeName does not implement doExecuteBroadcast")
   }
 
+  def defaultDatabaseNotExistsError(defaultDatabase: String): Throwable = {
+    new SparkDefaultDatabaseNotExistsException(
```

Comment:
We don't need a new exception. We can just use `SparkException`
```
new SparkException(
  errorClass = "DEFAULT_DATABASE_NOT_EXISTS",
  messageParameters = Map("defaultDatabase" -> defaultDatabase)
)
```

Reply:
Done